code/controllers/subsystem/garbage.dm

```diff
@@ -207,6 +210,14 @@ var/datum/subsystem/garbage_collector/SSgarbage
 		return TRUE
```

Comment:
Negative numbers are TRUE by the way. Anything other than 0, null, or "" is TRUE.